src/quo2/components/wallet/account_origin/view.cljs

```diff
@@ -0,0 +1,117 @@
+(ns quo2.components.wallet.account-origin.view
+  (:require [quo2.theme :as quo.theme]
+            [react-native.core :as rn]
+            [quo2.components.wallet.account-origin.style :as style]
+            [quo2.components.markdown.text :as text]
+            [quo2.components.icon :as icons]
+            [quo2.components.avatars.user-avatar.view :as user-avatar]
+            [utils.i18n :as i18n]
+            [quo2.foundations.colors :as colors]))
+
+(defn- row-title
+  [type user-name]
+  [text/text
+   {:weight :medium
+    :size   :paragraph-1}
+   (case type
```

Comment:
nitpick: I'd rename type since it shadows a Clojure core function.

Reply:
https://github.com/status-im/status-mobile/pull/17147#discussion_r1310321396